src/css/docs-prose.css

```diff
@@ -127,6 +127,16 @@
   }
 }
 
+a.btn.btn--blue {
+  color: #fff;
+  text-decoration: none;
+  margin: 18px 0;
+}
+
+a.btn.btn--blue:hover {
+  color: #fff;
+}
+
```

Comment:
This CSS will effect all elements with `btn btn--blue` classes. I think we'll want to be more specific here to prevent unintended side effects to other elements.

We have a couple options:

1. Prepend `.prose` to line 130 and 136. However this could effect other buttons.
2. Use passthroughProps in Button: `passthroughProps={{ download: fileName, className:'unprose btn btn--blue round-full py12 px24 txt-s my18' }}`. While we would need to rebuild the button classes to pass `unprose` to the button, we don't need to maintain CSS nor be concerned if the classes will effect other elements.

My preference is towards option 2, I like that we can maintain the styling for the component in one place and not rely CSS in docs-prose.css. What do you think?

Reply:
I am very much for option two as well so we don't have to worry about any unintended consequences in the future ✨  

> I like that we can maintain the styling for the component in one place and not rely CSS in docs-prose.css

💯